pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -781,9 +785,8 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 				return
 			}
 		} else {
-			// Default pod memory limit to the current memory limit if unset to prevent it from updating.
-			// TODO(#128675): This does not support removing limits.
-			podResources.Memory = currentPodMemoryConfig.Memory
+			value := maxValue
+			podResources.Memory = &value
```

Comment:
nit

```suggestion
			value := maxValue
			podResources.Memory = ptr.To(maxValue)
```

(import "k8s.io/utils/ptr")

same in other places

Reply:
Done. Thanks.